docs/WindowAggregation-sparkcompute.md

```diff
@@ -39,12 +39,41 @@ specifies the value to be added to the value of the current row to get the end b
 
 
 **Aggregates:** Specifies a list of functions to run on the selected window. Supported aggregate functions are `Rank`, 
-`Dense Rank`, `Percent Rank`, `N tile`, `Row Number`, `Median`, `Continuous Percentile`, `Lead`, 
-`Lag`, `First`, `Last`, `Cumulative distribution`, `Accumulate`. Aggregates are specified using syntax: 
+`Dense_Rank`, `Percent_Rank`, `N_Tile`, `Row_Number`, `Median`, `Continuous_Percentile`, `Lead`, 
```

Comment:
is this ALL_CAPS? It seems to be ALL_CAPS in the pipeline JSON.

Reply:
Function names are not case sensitive. 


https://github.com/AdaptiveScale/window-aggregation/blob/d95b30d5d11ace393a1dfe00e5400494f0a6dbab/src/main/java/io/cdap/plugin/WindowAggregationConfig.java#L222-L224